components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -762,16 +762,34 @@ IJsonSerializer jsonSerializer
         }
 
         /// <inheritdoc/>
-        public async Task<List<PublishedNodesEntryModel>> GetConfiguredEndpointsAsync(
+        public async Task<List<ConfiguredEndpointModel>> GetConfiguredEndpointsAsync(
```

Comment:
I think we could avoid duplicate code by just using  here with PublishedNodesEntryModel here. Of course with no OpcNodes set.